From fe4489c44971fa10f5cd48ec81aa7dc5b4f720e8 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Sat, 19 Jul 2025 12:27:48 +0200
Subject: [PATCH] Drop duplicate required KF6DocTools dependency

It is already being searched for prior to the main KF6 dependencies,
and the docs subdir is being added conditionally, so that will have
been the intention.

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
(cherry picked from commit ba806f96ff3081fe70f02f27b74ebd936ba2d0ad)
---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 47e67c1b..eef9a2b1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,7 +46,6 @@ find_package(KF6 ${KF_MIN_VERSION} REQUIRED COMPONENTS
     Config
     ConfigWidgets
     CoreAddons
-    DocTools
     I18n
     ItemViews
     KCMUtils
@@ -93,7 +92,6 @@ ecm_set_disabled_deprecation_versions(QT 5.15.2  KF 5.101.0)
 
 # subdirectories to build
 ecm_optional_add_subdirectory(data)
-ecm_optional_add_subdirectory(doc)
 ecm_optional_add_subdirectory(src)
 # ecm_optional_add_subdirectory(sounds)
 ecm_optional_add_subdirectory(icons)
@@ -101,6 +99,7 @@ ecm_optional_add_subdirectory(icons)
 # files to install in the ktouch project root directory
 ki18n_install(po)
 if (KF6DocTools_FOUND)
+    ecm_optional_add_subdirectory(doc)
     kdoctools_install(po)
 endif()
 install( PROGRAMS org.kde.ktouch.desktop  DESTINATION  ${KDE_INSTALL_APPDIR} )
-- 
2.51.0

